sdcm/tester.py

```diff
@@ -221,6 +221,17 @@ def setUp(self):
             self.db_cluster.wait_for_init()
         if self.cs_db_cluster:
             self.cs_db_cluster.wait_for_init()
+
+        # change RF of system_auth
+        system_auth_rf = self.params.get('system_auth_rf', default=3):
```

Comment:
bug!

Reply:
Do you expect the default value is set in sct_config.py?

Reply:
I saw you had backported PR https://github.com/scylladb/scylla-cluster-tests/pull/906 to branch-2019.1, thanks!